src/lib/shady-render.ts

```diff
@@ -275,12 +284,18 @@ export const render =
       if (firstScopeRender) {
         const part = parts.get(renderContainer)!;
         parts.delete(renderContainer);
-        if (part.value instanceof TemplateInstance) {
-          prepareTemplateStyles(
-              renderContainer as DocumentFragment,
-              part.value.template,
-              scopeName);
-        }
+        // ShadyCSS might have style sheets (e.g. from `prepareAdoptedCssText`)
+        // that should apply to `renderContainer` even if the rendered value is
+        // not a TemplateInstance. However, it will only insert scoped styles
+        // into the document if `prepareTemplateStyles` has already been called
+        // for the given scope name. `prepareTemplateStyles` requires a
+        // template element (and the local wrapper requires a Template), so we
+        // create an empty one here to satisfy it.
```

Comment:
Update this comment since we're not creating an empty template here.

Reply:
Updated.